Makefile

```diff
@@ -12,7 +12,7 @@ endif
 GO_TAGS := netgo
 LD_FLAGS := -s -w -X github.com/gptscript-ai/knowledge/version.Version=${GIT_TAG}
 build:
-	CGO_ENABLED=0 go build -o bin/knowledge -tags "${GO_TAGS}" -ldflags '$(LD_FLAGS)' .
+	go build -o bin/knowledge -tags "${GO_TAGS}" -ldflags '$(LD_FLAGS)' .
```

Comment:
Are we certain that turning on CGo here doesn't cause any issues on each OS?

Reply:
Yeah, tbh I haven't tested on other OS, but the c library is linked from go module. Will test on other os too 👍